plugins/flowdock/lib/samson_flowdock/samson_plugin.rb

```diff
@@ -0,0 +1,43 @@
+module SamsonFlowDock
+  class Engine < Rails::Engine
+  end
+end
+
+Samson::Hooks.callback :stage_defined do
+  Stage.class_eval do
+    has_many :flowdock_flows
```

Comment:
since this will be the canonical example, can we work out where the migration will live and how it will run?